src/components/displaySettings/displaySettings.template.html

```diff
@@ -77,23 +93,29 @@ <h2 class="sectionTitle">
             <option value="nl">Dutch</option>
             <option value="en-GB">English (United Kingdom)</option>
             <option value="en-US">English (United States)</option>
+            <option value="eo">Esperanto</option>
+            <option value="fil">Filipino</option>
             <option value="fi">Finnish</option>
             <option value="fr">French</option>
             <option value="fr-CA">French (Canada)</option>
+            <option value="gl">Galician</option>
             <option value="de">German</option>
             <option value="el">Greek</option>
             <option value="he">Hebrew</option>
-            <option value="hi-IN">Hindi (India)</option>
+            <option value="hi-IN">Hindi</option>
```

Comment:
I assume the suffix is applied to any code with a dash, so this makes sense to me. Why remove it?

Reply:
To match what is on Weblate. We could split it and add normal Hindi, not sure if Weblate would then re-add the parenthesis

www.lingoes.net/en/translator/langcode.htm

Reply:
There isn't any other language codes for Hindi so i'm not sure if we should even add it as I think the India got appended because someone selected hi-IN instead of hi

Reply:
Matching weblate is a good reason, sounds good.